invenio_cli/cli/packages.py

```diff
@@ -67,11 +67,26 @@ def install(cli_config, packages, skip_build):
 @pass_cli_config
 def outdated(cli_config):
     """Show outdated Python dependencies."""
-    raise click.UsageError("Not supported yet.")
+    steps = PackagesCommands.outdated_packages()
 
+    on_fail = ""
```

Comment:
major: This two string should contain a value... otherwise the user will not understand....

Reply:
The string would be redundant with the information already displayed by the command itself, it would be redundant.

Reply:
I understand, but is more future proof (i.e. in case it becomes more than one step)...